source/comInterfaces_sconscript

```diff
@@ -26,18 +30,62 @@ def new_my_import(fullname):
 	return old_my_import(fullname)
 comtypes.client._generate._my_import = new_my_import
 
-def interfaceAction(target,source,env):
-	clsid=env.get('clsid')
+def makeIDEFriendly(path:str) -> None:
+	"""
+	Add a local import of * so that tools and IDE's can find definitions.
+	Prefer to import from comtypes.gen, at runtime behavior will not have changed.
+	Make code like:
+	``` python
+	from comtypes.gen import _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0
+	globals().update(_C523F390_9C83_11D3_9094_00104BD0D535_0_3_0.__dict__)
+	__name__ = 'comtypes.gen.AcrobatAccessLib'
+	```
+	Become:
+	``` python
+	try:
+		from comtypes.gen import _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0
+	except ModuleNotFoundError:
+		from _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0 import *
+	globals().update(_C523F390_9C83_11D3_9094_00104BD0D535_0_3_0.__dict__)
+	__name__ = 'comtypes.gen.AcrobatAccessLib'
+	```
+	@param path: Path to the friendly name comInterfaces module.
+	"""
+	importTemplate = (
+"""try:
+	from comtypes.gen import {libIdentifier}
+except ModuleNotFoundError:
+	import {libIdentifier}
+	from {libIdentifier} import *
+""")
+
+	importPattern = re.compile(r"from comtypes\.gen import ([\w]+)\n")
+	for line in fileinput.input(path, inplace=True):

```

Comment:
A comment here mentioning that fileinput.Input temporarily replaces sys.stdout would be useful.

Reply:
Ah yes, you are right. It is an unintuitive library.